syft/workers/base.py

```diff
@@ -117,6 +119,21 @@ def __init__(
                         if self.id not in worker._known_workers:
                             worker.add_worker(self)
 
+    def remove_worker_from_registry(self, worker_id):
+        """remove a worker from the dictionary of known workers
+
+        :param worker_id: id to be removed
```

Comment:
change these to [google docstring style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)